Issue 126

Issamricin opened the pull request:
Main points:
- Created a new class called acceleration that takes care of drawing the 3d acceleration and it's data.
- Integrated this class into the cli class, now the user will also be prompted to enter a vector(x,y,z) data through the cli just as the azimuth and the other data.
- Created a signal manager class that takes signal from the acceleration class and send it to the compass class to be view in the information section acceleration: (x,y,z).
- Integrated the new acceleration into the simulator and simulated data for the acceleration.
- Added several labs under the lab/matplotlib

alanmehio commented:
Good job. Just implement the comment and all will be fine for next release. The README.rst  should not contain any broken links so you need to double check after you release to production.

alanmehio reviewed (commented):
I am sorry. I need to close my review which I did not do.
Now you should be able to see my review. 
I have started the review but I have never submitted my code review. You see
My comment on PR is visible to you but my code comments are not until I submit my review to your . Sorry

Issamricin commented:
changed both the file name and class to accele3D_signal_manager.py and class name accelerationSignalManager.
And removed underscore from Acceleration_3D class name -> Acceleration3D
